.bumpversion.cfg

```diff
@@ -0,0 +1,12 @@
+[bumpversion]
+current_version = 2.18.1
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+
+serialize = 
+	{major}.{minor}.{patch}
+
+[bumpversion:file:lib/recurly/version.rb]
+
+[bumpversion:file:README.md]
```

Comment:
Nit: you might want to change your editor to make sure there is a newline at the end of every file. That's what the red circle is. It's not good practice to have an EOF on a non-empty line.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file
https://github.com/Microsoft/vscode/issues/12076#issuecomment-340037747

Reply:
Done!

<img width="1020" alt="Screen Shot 2019-09-18 at 3 35 52 PM" src="https://user-images.githubusercontent.com/15930699/65183948-2245eb80-da2a-11e9-8d4b-04893fae7add.png">